lib/neuron.ex

```diff
@@ -116,6 +116,11 @@ defmodule Neuron do
     Map.put(body, :variables, variables)
   end
 
+  defp handle_response(response, options) do
+    parse_options = parse_options(options)
```

Comment:
Could we change this variable name to `parsed_options` to differentiate it from the function?